### ISSUE TITLE
Not showing complete results

### ISSUE THREAD

**Issue by shawn1thompson** [gjedeer/Acastus#7]
When I search for Starbucks, for example, it does some locations near me, but then jumps to a completely different city making it difficult to find the one I'm actually trying to get to. I can provide a screenshot if you need. I'm doing this from my phone and don't see a way to.

**Comment by gjedeer** [comment]
Hi,

Thank you for the report. Are you seeing the same with raw Photon? https://photon.komoot.de/

If yes: you should probably file a bug here https://github.com/komoot/photon/issues

**Comment by shawn1thompson** [comment]
Hi,
I don't see how to search from a specific reference point, i.e., my location. i tried using https://photon.komoot.de/?q=indianapolis in the address bar, but it returned the same results as the default URL. So, I'm not sure how to test using the website.
Thanks.

**Comment by k3b** [comment]
> ... [ I ] don't see a way to.

Since there are to many hits when searching for "starbucks" (worldwide) you can add the city you are in to your search (i.e. "starbucks berlin" assumig that you are in berlin.)

This works pretty well for me.

**Comment by shawn1thompson** [comment]
That does work somewhat better. I'm looking at this as someone who doesn't know where a Starbucks is. Since the app knows my location, shouldn't it give me all locations progressively closest to farthest away from me? Entering Starbucks Indianapolis does somewhat better, but it still jumps as you can see in the screenshot I've attached. There are many locations closer than the one listed at 138 miles away. Also, the list doesn't scroll. The last entry is the one in the screen, i.e., the one at 138 miles away.

Thanks again.

Shawn Thompson
shawn.thompson@politicalphilosopher.xyz
"Each one teach one"

-------- Original Message --------

**Comment by gjedeer** [comment]
Hmm, I think it's the backend. When I'm searching for Starbucks in my area, I'm getting listings from my city and then the next listings are some 5700 km away. 

I need to review if the app waits for GPS fix or network location before it sends the query, because once I only had the results from 5700 km away.

Hehe, the Starbucks in La Cité-Limoilou, Canada must me a very good one because it also comes up in each and every one of my searches.

**Comment by gjedeer** [comment]
Let's see what the Photon dev has to say

**Comment by shawn1thompson** [comment]
It would definitely be good to verify whether the app has location or not. I had assumed the location icon next to the text field was the location though. Maybe not?

**Comment by gjedeer** [comment]
We got an answer, not much different from what you two have figured out:

https://github.com/komoot/photon/issues/375

**Comment by gjedeer** [comment]
Tagged a new version 1.16, to be found in F-Droid soon.

Thanks to a great suggestion from @lonvia I've added a "location bias" parameter when asking Photon about the location, which has greatly improved the results.

I've also added a location icon per @shawn1thompson suggestion. The icon itself is, errrr, not completely relevant to what it does, but I couldn't find a better one under a free license. Suggestions or pull requests to change it are welcome.
